genomics/01-bap_resequencing.Rmd

```diff
@@ -14,11 +14,57 @@ output:
 ## Accessing the data
 
 ```{r}
+key <- read.table('/data/terraref/sites/genomics/raw_data/ril/gbs/Key_ril_terra')
+head(key)
+```
+
 
-readLines('/data/terraref/sites/genomics/raw_data/ril/gbs/Key_ril_terra')
+```{r}
+key <- read.table("/data/terraref/sites/genomics/raw_data/ril/gbs/Key_ril_terra",
+                  header = TRUE)
+key$LibraryPrepID <- seq(1:nrow(key))
+key$Enzyme <- rep("ApeKI", times = nrow(key))
+key$FullSampleName <- paste0(as.character(key$Sample), ":", 
+                             as.character(key$LibraryPrepID))
+names(key) <- c("Flowcell", "Lane", "Barcode", "DNASample", "LibraryPlate", 
+                "Row", "Col", "LibraryPrepID", "Enzyme")
+write.table(key, "/home/rstudio/tutorials/genomics/key_v5",
+            quote = FALSE, row.names = FALSE, sep = "\t")
+```
 
+# index genome
+```{bash}
+cd tutorials
+mkdir ref_genome
+git clone https://github.com/lh3/bwa.git
```

Comment:
could this be replaced by `apt-get install bwa`?

Reply:
I don't think this can be replaced by "apt-get install bwa".  BWA is a bioinformatics package for alignment.

Reply:
@dlebauer @zhenbinHU look at the following. I'd structure it similarly...
 https://github.com/terraref/documentation/blob/master/genomic-data-product.md